### PR TITLE
Station list card improvements for health metrics

### PR DESCRIPTION
**Pull request by pantelisss**
## **Why?**
Health metrics in Home Screen 
### **How?**
- Integrated `Metrics` model in all device models
- Updated weather area layout in the devices list
- Removed station indication "overflow layout" from card items
### **Testing**
Make sure the card items are rendered as expected.
You can "play" with the mock scheme and `get_user_devices` json
### **Additional context**
fixes fe-1187

## Summary by CodeRabbit

- **New Features**
	- Expanded font icon options with new icons for enhanced visual representation.
	- Introduced a new display mode, `.grid`, in the Weather Overview View for improved layout flexibility.
	- Added new properties to enhance device metrics and status tracking.
	- New computed properties for device quality status and weather fields.
	- Enhanced layout capabilities with new spacing options in UI components.
	- Added new localization cases for improved user experience regarding data availability and station status.
	- Improved representation of issues in the Station Chips View with a structured format.

- **Bug Fixes**
	- Updated localization strings for better user experience in weather-related displays.

- **Documentation**
	- Enhanced localization support with new string entries for better accessibility.

- **Chores**
	- Streamlined initialization processes for various UI components to improve code maintainability.
	- Simplified logic for displaying device indications, enhancing clarity.

### CODE REVIEW

**Review by PavlosTze — CHANGES_REQUESTED** [review]
Except my comment below I have found 1 more issue:
1. When I get `NO_LOCATION_DATA` I would expect next to the address icon to see the "No Location" label but I see the exact address "Chania, GR" for example in `High Denim Ice`.

File: PresentationLayer/Extensions/Numeric/Int+.swift [hunk 1]
```diff
@@ -66,4 +66,17 @@ extension Int {
...
+			case 0..<10:
+				return .error
+			case 10..<95:
+				return .warning
+			case 95...100:
+				return nil
+			default:
+				return nil
+		}
+	}
```
[the remark this answers is not in the document]
Comment (pantelisss) [reply PresentationLayer/Extensions/Numeric/Int+.swift:81]: Will change in an upcoming PR

**Comment by pantelisss** [comment]
> Except my comment below I have found 1 more issue:
> 
> 1. When I get `NO_LOCATION_DATA` I would expect next to the address icon to see the "No Location" label but I see the exact address "Chania, GR" for example in `High Denim Ice`.

I've made a question about this in the feature channel

**Comment by pantelisss** [comment]
@PavlosTze I think we are ready to move on

**Review by PavlosTze — APPROVED** [review]

**Pull request closed without merging**